README.fr.md

```diff
@@ -0,0 +1,122 @@
+[![npm version](https://badge.fury.io/js/name-suggestion-index.svg)](https://badge.fury.io/js/name-suggestion-index)
+
+## nom-suggestion-index 
+#### TODO: nom-suggestion-index or index de nom-suggestion?
+Noms de marque communs canoniques pour OpenStreetMap.
+
+
+### Qu'est-ce que c'est?
+
+Le but de ce projet est de maintenir un [canonique](https://fr.wikipedia.org/wiki/Canonique_(informatique))
+liste des noms couramment utilisés pour suggérer l'orthographe et le balisage cohérents des fonctionnalités
+dans OpenStreetMap.
+
+[Regardez la vidéo](https://2019.stateofthemap.us/program/sat/mapping-brands-with-the-name-suggestion-index.html) de notre conférence à State of the Map US 2019 pour en savoir plus sur ce projet!
+
+
+### Parcourir l'index
+
+Vous pouvez parcourir l'index à https://nsi.guide/ pour voir quelles marques manquent de liens Wikidata ou ont des pages Wikipedia incomplètes.
+
+
+### Comment il est utilisé
+
+Lorsque les mappeurs créent des fonctionnalités dans OpenStreetMap, ils ne sont pas toujours cohérents sur la façon dont ils
+nommez et étiquetez les choses. Par exemple, nous pouvons préférer `McDonald's` étiqueté comme` amenity = fast_food`
+mais nous voyons de nombreux exemples d'autres orthographes («Mc Donald's», «McDonalds», «McDonald's») et
+étiquetages (`amenity = restaurant`).
+
+La construction d'un index de nom canonique permet deux choses très utiles:
+- Nous pouvons suggérer la manière la plus "correcte" d'étiqueter les choses lorsque les utilisateurs les créent pendant l'édition.
+- Nous pouvons analyser les données OSM pour les fonctionnalités "incorrectes" et produire des listes pour révision et nettoyage.
+
+<img width="1017px" alt="Index de suggestion de noms utilisé dans iD" src="https://raw.githubusercontent.com/osmlab/name-suggestion-index/master/docs/img/nsi-in-iD.gif"/>
+
+* L'index de suggestion de nom est utilisé dans iD lors de l'ajout d'un nouvel élément *
+
+Actuellement utilisé dans:
+* iD (voir ci-dessus)
+* [Vespucci](http://vespucci.io/tutorials/name_suggestions/)
+* Présélections JOSM disponibles
+
+
+### Participer!
+
+* Lisez le projet [Code de conduite](CODE_OF_CONDUCT.md) et n'oubliez pas d'être gentils les uns envers les autres.
+* Voir [CONTRIBUTING.md](CONTRIBUTING.md) pour savoir comment contribuer à cet index.
+
+Nous cherchons toujours de l'aide! Si vous avez des questions ou souhaitez contacter un responsable, envoyez une requête ping à `bhousel` sur:
+* [OpenStreetMap US Slack](https://slack.openstreetmap.us/)
+(Chaînes `# poi` ou` # general`)
+
+
+### Conditions préalables
+
+* [Node.js](https://nodejs.org/) version 8 ou plus récente
+* [`git`](https://www.atlassian.com/git/tutorials/install-git/) pour votre plateforme
+
+
+### Installation
+
+* Clonez ce projet, par exemple:
+  `git clone git@github.com: osmlab / nom-suggestion-index.git»
+* `cd` dans le dossier du projet,
+* Exécutez `npm install` pour installer les bibliothèques
+### À propos de l'index
```

Comment:
Might need to preserve the spacing here and below for markdown to continue to work.